tools/clang/test/HLSLFileCheck/hlsl/types/modifiers/reordercoherent/reordercoherent2.hlsl

```diff
@@ -0,0 +1,17 @@
+// RUN: %dxc -E main -T lib_6_9 %s | FileCheck %s
+
+// CHECK: 'reordercoherent' is not a valid modifier for a declaration of type 'float'
```

Comment:
This test is already covered by `tools/clang/test/SemaHLSL/reordercoherent-type-errors.hlsl`.

Reply:
Removed redundant test